src/backend/storeManagers/nile/library.ts

```diff
@@ -75,11 +81,14 @@ function loadGamesInAccount() {
       install: info
         ? {
             install_path: info.path,
+            // For some time size was undefined in installed.json, that's why we
+            // need to keep this fallback to 0
+            install_size: getFileSize(info?.size ?? 0),
```

Comment:
`info` is already known to not be `undefined`/`null` here
```suggestion
            install_size: getFileSize(info.size ?? 0),
```